move/interchain_token_service/sources/types/token_manager_types.move

```diff
@@ -0,0 +1,24 @@
+module interchain_token_service::token_manager_types {
+    // === Types ===
+    public enum TokenManagerType {
+        NATIVE_INTERCHAIN_TOKEN, // This type is reserved for interchain tokens deployed by ITS, and can't be used by custom token managers.
+        LOCK_UNLOCK, // The token will be locked/unlocked at the token manager.
+        MINT_BURN, // The token will be minted/burned on transfers. The token needs to give mint and burn permission to the token manager.
```

Comment:
enum needs to exactly match the original definition
https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol#L10